### PR TITLE
feat(cb2-8133): stop disabled accounts going into reference data

### PR DESCRIPTION
**Pull request by naathanbrown**
## Ticket title
Update the API call to only get back accounts with accountEnabled set to true
[https://dvsa.atlassian.net/browse/CB2-8133](8133)

## Checklist
- [ ] Code has been tested manually
- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/aad/getMemberDetails.ts [hunk 1]
```diff
@@ -22,19 +22,25 @@ axios.interceptors.response.use((response) => response, onRejected);
 export const getMemberDetails = async (): Promise<IMemberDetails[]> => {
   const aadBase = config.aad.baseUrl;
   const groupIds = config.aad.groupId.includes(',') ? config.aad.groupId.split(',') : [config.aad.groupId];
-  const membersToRequest = config.aad.membersToRequest;
+  // const membersToRequest = config.aad.membersToRequest;
```
Comment (gjulien-bjss) [inline src/aad/getMemberDetails.ts:25]:
```suggestion
```

File: src/aad/getMemberDetails.ts [hunk 2]
```diff
@@ -22,19 +22,25 @@ axios.interceptors.response.use((response) => response, onRejected);
...
     logger.info(`Trying to get aad member list for group: ${groupId.trim()}`);
 
     // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
     axios.defaults.headers.common.Authorization = `Bearer ${accessToken}`;
 
-    const response = await axios.get<MemberList>(requestUrl, { headers: { Authorization: `Bearer ${accessToken}` } });
+    const response = await axios.get<MemberList>(requestUrl, {
+      headers: { Authorization: `Bearer ${accessToken}`, ConsistencyLevel: 'eventual' },
+    });
+    console.log(response.data.value);
```
Comment (gjulien-bjss) [inline src/aad/getMemberDetails.ts:43]:
```suggestion
```
not sure if intentional

Comment (gjulien-bjss) [reply src/aad/getMemberDetails.ts:43]: Or use the `logger`

File: src/aad/getMemberDetails.ts (hunk 1 above)
Comment (naathanbrown) [reply src/aad/getMemberDetails.ts:25]: MY BAD

File: src/aad/getMemberDetails.ts (hunk 2 above)
Comment (naathanbrown) [reply src/aad/getMemberDetails.ts:43]: I have had a shocker here

**Review by shivangidas — APPROVED** [review]

**Pull request merged**